core/mono-samples/iOS/README.md

```diff
@@ -0,0 +1,32 @@
+---
+languages:
+- csharp
+products:
+- dotnet-core
+page_type: sample
+name: "iOS Sample: Simple greeting and counter (C#)"
+description: "An iOS application that contains an example of embedding the mono runtime to invoke unmanaged code with C#."
+urlFragment: "mono-ios-csharp"
+---
+
+# iOS Sample: Simple greeting and counter (C#)
+
+In this sample, the mono runtime is used to invoke Objective-c unmanaged code (main.m) from the C# managed side (Program.cs) and vice versa. With the sample running, you can enter your name and click the corresponding button to modify the greeting message as well as clicking a button to increment a counter.
+
+[!NOTE]
+The purpose of this sample is to demonstrate the concept of building an iOS application on top of the mono runtime. The mono runtime headers should be supplied through the build process.
+
+## Sample Prerequisites
+
+This sample will only run on macOS as it requires Xcode and an iOS simulator.
+
+- Xcode: Any version should work with this sample (download Xcode at <https://developer.apple.com/xcode/>).
+- iOS simulator 8.0 or greater.
+- Daily master build of the dotnet sdk (<https://github.com/dotnet/installer>).
```

Comment:
@steveisok @mdh1418 @akoeplinger 
Considering it's not easy to install the .NET 6 SDK at the moment, I would improve this with the specific commands to install. I don't see anyone figuring out how to install. I couldn't and had to have @steveisok help.

Reply:
Added more details to install the specific dotnet version. Was this what you meant?